cohort-cli/src/main/java/com/ibm/cohort/cli/input/MeasureContextProvider.java

```diff
@@ -34,6 +46,6 @@
 			parsedParameters = ParameterHelper.parseParameterArguments(parameters);
 		}
 
-		return Collections.singletonList(new MeasureContext(resourceId, parsedParameters));
+		return Collections.singletonList(new MeasureContext(resourceId, parsedParameters, null, null));
```

Comment:
would prefer an overloaded constructor to be added to `MeasureContext`

```suggestion
		return Collections.singletonList(new MeasureContext(resourceId, parsedParameters));
```